docker/config/test.env

```diff
@@ -17,3 +17,6 @@ database_hostname=postgresql
 database_port=5432
 database_username=postgres
 database_password=aPassword
+
+# Django tests are meant to run with debug mode disabled.
+DEBUG=False
```

Comment:
Interestingly, we have two sets of test variables. Some defined here and some defined in `setup.cfg` and `webapp-django/pytest.ini`. This file predates switching to pytest. I think we should probably move these vars to the pytest place and nix this file.

I wrote up https://bugzilla.mozilla.org/show_bug.cgi?id=1417913 to think about that.